Makefile.am

```diff
@@ -6584,6 +6586,11 @@ SYSINIT_TARGET_WANTS += \
 	systemd-sysctl.service \
 	systemd-ask-password-console.path
 
+if ENABLE_MACHINED
+MACHINES_TARGET_WANTS += \
+	var-lib-machines.automount
+endif
```

Comment:
I didn't change this as per my reasoning above. Please indicate if you still want this changed.